### ISSUE TITLE
Error while parsing .eslintrc

### ISSUE THREAD

**Issue by Vitaliy223** [eslint/eslint#9635]
I'm using eslinter in Atom.
This is error desciption:
Error: end of the stream or a document separator is expected at line 2, column 9:
module: {
            ^
This is my .eslintrc
```
module.exports = {
  module: {
    extends: "eslint:recommended",
    rules: {
          indent: ["error", 4],
          linebreak-style: ["error", "unix"],
          quotes: ["error", "double"],
          semi: ["error", "always"],

          comma-dangle: ["error", "always"],
          no-cond-assign: ["error", "always"],

          no-console: "off",},

    env: {
          node: true,
          es6: true,
      },
    }
}
```
I've used quotes, but nothing happened.

**Comment by platinumazure** [comment]
You have an extra `module` key in your config which I'm not sure ESLint would know how to parse. Try:

```js
module.exports = {
  extends: "eslint:recommended",
  rules: {
    indent: ["error", 4],
    linebreak-style: ["error", "unix"],
    quotes: ["error", "double"],
    semi: ["error", "always"],
    comma-dangle: ["error", "always"],
    no-cond-assign: ["error", "always"],
    no-console: "off"
  },
  env: {
    node: true,
    es6: true
  }
}
```

(Note: I removed the trailing commas because I wasn't sure if your environment supports them- if so, feel free to add them back.)

**Comment by JPeer264** [comment]
@platinumazure I think that won't be a valid object as `linebreak-style` must be wrapped in quotes `'linebreak-style'`. Following would be valid:

```js
module.exports = {
  extends: "eslint:recommended",
  rules: {
    indent: ["error", 4],
    "linebreak-style": ["error", "unix"],
    quotes: ["error", "double"],
    semi: ["error", "always"],
    "comma-dangle": ["error", "always"],
    "no-cond-assign": ["error", "always"],
    "no-console": "off"
  },
  env: {
    node: true,
    es6: true
  }
}
```

**Comment by ilyavolodin** [comment]
Closing as the question has been answered.
